### ISSUE TITLE
Step 4 error

### ISSUE THREAD

**Issue by alhsu** [sigven/pcgr#9]
hi Sigve,

PCGR ran into an error when generating output file, it may be a pcgr-writer or a deconstructSig problem. The error message is:
```2017-07-18 02:00:43 - pcgr-writer - INFO - STEP 4: Generation of output files
2017-07-18 02:01:09 [INFO] Reading and parsing VEP/vcfanno-annotated VCF file - /workdir/output/WES012germline.pcgr.vcf.gz
2017-07-18 02:02:29 [INFO] Number of PASS variants: 38715
2017-07-18 02:02:30 [INFO] Number of SNVs: 36991
2017-07-18 02:02:30 [INFO] Number of deletions: 801
2017-07-18 02:02:30 [INFO] Number of insertions 917
2017-07-18 02:02:30 [INFO] Extending annotation descriptions related to PFAM protein domains
2017-07-18 02:02:32 [INFO] Extending annotation descriptions related to UniprotKB/SwissProt protein features
2017-07-18 02:02:40 [INFO] Extending annotation descriptions related to Database of Curated Mutations (DoCM)
2017-07-18 02:02:44 [INFO] Extending annotation descriptions related to ClinVar
2017-07-18 02:02:55 [INFO] Generating data for tiered cancer genome report
2017-07-18 02:02:55 [INFO] Number of coding variants: 12105
2017-07-18 02:02:56 [INFO] Number of noncoding variants: 26612
2017-07-18 02:02:58 [INFO] Identifying weighted contributions of known mutational signatures using deconstructSigs (n = 36993 SNVs)
2017-07-18 02:03:01 [INFO] Inferred weighted contribution of S1: 0.708
2017-07-18 02:03:01 [INFO] Inferred weighted contribution of S12: 0.224
Error in rbind(deparse.level, ...) :
  numbers of columns of arguments do not match
Calls: <Anonymous> -> <Anonymous> -> <Anonymous> -> rbind -> rbind
Execution halted
```
Would you be able to help figuring out how this error occurred?

best,
Arthur

**Comment by sigven** [comment]
Hi @alhsu,
Thanks for your message, and apologees for the late response. I am working on a new release, and would really appreciate to debug your problem. Could you send me your input VCF through email (sigven AT ifi.uio.no)?

best,
Sigve

**Comment by alhsu** [comment]
hi Sigve,

I've sent you the VCF that caused the issue. Please let me know if they didn't get sent properly.

best,
Arthur

**Comment by sigven** [comment]
Fixed in 0.4.0 release
